smartsim/experiment.py

```diff
@@ -151,19 +151,23 @@ def __init__(self, name: str, exp_path: str | None = None):
         experiment
         """
 
-    def start(self, *jobs: Job) -> tuple[LaunchedJobID, ...]:
+    def start(self, *jobs: Job | t.Sequence[Job]) -> tuple[LaunchedJobID, ...]:
         """Execute a collection of `Job` instances.
 
         :param jobs: A collection of other job instances to start
         :returns: A sequence of ids with order corresponding to the sequence of
             jobs that can be used to query or alter the status of that
             particular execution of the job.
         """
+        # If item is instance iterable then unpack and extend the list
+
+        jobs_ = list(tuple(_helpers.unpack(jobs)))
```

Comment:
No need to cast an iterable to a `tuple` and then to a `list`. Consider just casting to a `list` directly!
```suggestion
        jobs_ = list(_helpers.unpack(jobs))
```